allure-report-face/src/util/attachmentType.js

```diff
@@ -24,6 +24,8 @@ export default function typeByMime(type) {
             return 'html';
         case 'text/csv':
             return 'csv';
+        case 'text/tab-separated-values':
```

Comment:
Why you can't use the same type for this type? I mean `attachmentType.js` should provide view type for given media type. I guess it should return `table` for both 'text/csv' and 'text/tab-separated-values'. That will solve all your problems as well